test/puppetlabs/puppetdb/http/events_test.clj

```diff
@@ -349,7 +349,7 @@
        :new_value nil
        :containing_class "Foo"
        :report_receive_time "2014-04-16T12:44:40.978Z"
-       :report "99ec099bed6dfb9bff2c7df7828270e95f590147"
+       :report "7d0cfa08901e1e1d80cf2f2f814d356d0e457e09"
```

Comment:
Did we see the hash change because we were serializing the date to JSON before computing the hash? And thus truncating the milliseconds before computing the hash?